.github/workflows/ci.yml

```diff
@@ -53,8 +44,40 @@ jobs:
         run: sudo apt update && sudo apt install libxml2-utils
       - name: Set project version
         run: echo "PROJECT_VERSION=$(xmllint --xpath '/*[local-name()="project"]/*[local-name()="version"]/text()' pom.xml)" >> $GITHUB_ENV
-      - name: Publish to the Maven Central Repository
-        run: if [[ "$PROJECT_VERSION" =~ .*SNAPSHOT ]]; then mvn -B deploy; fi
+      # Publish snapshot
+      - name: Set up JDK 17 for publishing a snapshot
+        if: github.event_name == 'push' && endswith(env.PROJECT_VERSION, 'SNAPSHOT')
+        uses: actions/setup-java@v3
```

Comment:
Hier und in Zeile 67 ebenfalls die aktuellste `v4` Version nehmen :)

Reply:
:+1: